faq.html

```diff
@@ -95,7 +95,7 @@ <h4 class="panel-title">
             <div class="panel panel-default">
               <div class="panel-heading">
                 <h4 class="panel-title">
-                  <a class="accordion-toggle collapsed" data-toggle="collapse" data-parent="#accordion" href="#collapseTwo">Audio is playing way too fast!</a>
+                  <a class="accordion-toggle collapsed" data-toggle="collapse" data-parent="#accordion" href="#collapseTwo">The audio is playing way too fast!</a>
```

Comment:
I'm surprised, but it seems you're right. 'Audio' [apparently](https://dictionary.cambridge.org/dictionary/english/audio) is an adjective (so can't have an article), but it also means that the way you used it isn't correct either (you used it as a noun).